kubernetes/jasperreports-server-service-deployment.yaml

```diff
@@ -21,12 +21,12 @@ metadata:
   namespace: jaspersoft
 spec:
   ports:
-  - name: http
-    port: 8080
-    targetPort: 8080
-  - name: https
-    port: 8443
-    targetPort: 8443
+    - name: http
+      port: 80
```

Comment:
I am curious why the move to standard ports? Given these serve as examples I would prefer to use secondary options like 8080 and 8443, when a user is ready to move from testing to production, they can modify accordingly. (In many cases these standard ports may be occupied by preexisting web-servers) 

Thoughts?

Reply:
I tried to keep the 8080/8443 port but it was failed with liveness probe and readiness probe due to security group restrictions on load balancer side in any cloud provider because it will deny the traffic by default , unless will allow the traffic manually and  till then it was keep on failing and Jrs never come up . That's why I made these changes to pass the all the health checks.

Reply:
OK, sounds good.